### DISCUSSION TITLE
2008/01/02/vi-server-%EF%BC%88vi-%E6%9C%8D%E5%8A%A1%E5%99%A8%EF%BC%89

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#201] [Announcements]
# 2008/01/02/vi-server-%EF%BC%88vi-%E6%9C%8D%E5%8A%A1%E5%99%A8%EF%BC%89

VI Server 是指通过程序调用 LabVIEW 提供的一些服务功能的技术。使用 VI Server 可以完成如下的功能：运行远程的VI程序；通过互联网运行程序、传输数据；程序运行时调整VI、控件的某些属性，主要用来调整界面；编程来创建或修改VI；搭建插件框架式程序，等等。

https://blog.qizhen.xyz/2008/01/02/vi-server-%EF%BC%88vi-%E6%9C%8D%E5%8A%A1%E5%99%A8%EF%BC%89

**Comment by ruanqizhen** [comment]
之前博客下的留言：

小西
2008年05月14日 17:36 编辑
路过~~~

回复
永青
2008年01月3日 09:10 编辑
http://yeyongq.spaces.live.com/blog/cns!A52629657552983C!914.entry

回复
